### PR TITLE
silenced pyICU install warning

### LINKED ISSUE #2990: Silence noisy warnings for pyicu
See this when running SCTK latest develop:
``
scancode-toolkit-master/venv/lib/python3.6/site-packages/fingerprints/cleanup.py:54: ICUWarning: Install 'pyicu' for better text transliteration.
  text = ascii_text(text)
``

### PR DESCRIPTION
**Pull request by rritik772**
# Silence noisy warnings for pyicu #2990

Fixes #2990 

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. (Getting segment fault and auto signout of linux account)
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

Signed-off-by: Ritik Ranajn <rritik772@outlook.com>

### CODE REVIEW

File: src/summarycode/summarizer.py [hunk 1]
```diff
@@ -161,6 +162,8 @@ def get_declared_holders(codebase, holders_tallies):
     A declared holder is a copyright holder present in the key files who has the
     highest amount of refrences throughout the codebase.
     """
+    warnings.simplefilter('ignore', UnicodeWarning)
```
Comment (AyanSinhaMahapatra) [inline src/summarycode/summarizer.py:165]: Did this work for you?

Can you reproduce this warning showing up and then after adding this bit of code verify that it went away?

I tried and for me it didn't went away.

```
main_repos/gh-cli-repos/scancode-toolkit  silence-pyicu-warning ✔                                                                                                                                         1d13h  ⍉
▶ scancode -clipeu --classify --summary --license-clarity-score --license-text ../../../write_access/scancode-extra/tests/formattedcode/data/yaml/with-package-and-license --yaml output.yaml
Setup plugins...
Collect file inventory...
Scan files for: info, packages, licenses, copyrights, emails, urls with 1 process(es)...
[####################] 8
/home/ayansinha/nexB/main_repos/gh-cli-repos/scancode-toolkit/venv/lib/python3.8/site-packages/fingerprints/cleanup.py:54: ICUWarning: Install 'pyicu' for better text transliteration.
  text = ascii_text(text)
Scanning done.
Summary:        info, packages, licenses, copyrights, emails, urls with 1 process(es)
Errors count:   0
Scan Speed:     5.22 files/sec. 14.46 KB/sec.
Initial counts: 5 resource(s): 4 file(s) and 1 directorie(s)
Final counts:   5 resource(s): 4 file(s) and 1 directorie(s) for 11.09 KB
Timings:
  scan_start: 2023-01-30T195301.642569
  scan_end:   2023-01-30T195303.761515
  setup_scan:licenses: 1.29s
  setup: 1.29s
  scan: 0.77s
  total: 2.17s
Removing temporary files...done.

```

For me I had to set this in the main file, and not inside a function.

**Review by AyanSinhaMahapatra — CHANGES_REQUESTED** [review]
See comment inline:

1. Please make sure you can reproduce and can confirm that the fix works.
2. Maybe we should add a test that checks the console output for warnings, so atleast something in CI fails when these come up and we're notified?

**Comment by AyanSinhaMahapatra** [comment]
@rritik772 gentle ping!

**Comment by pombredanne** [comment]
@rritik772 ping!

**Comment by AyanSinhaMahapatra** [comment]
Closing this because of inactivity and in favour of #3280.

**Pull request closed without merging**